include/gauge_tools.h

```diff
@@ -169,4 +169,13 @@ namespace quda
   */
   void computeQChargeDensity(double energy[3], double &qcharge, void *qdensity, const GaugeField &Fmunu);
 
+  /**
+   * @brief Compute the trace of the polyakov loop in a given dimension
+   * @param[out] ploop The real adnd imaginary parts of the polyakov loop
```

Comment:
typo (adnd -> and)

Reply:
....and I guess it should be Polyakov (not polyakov)

Reply:
Done locally, will push soon

Reply:
https://github.com/lattice/quda/commit/4a893d83c4c5b4c15c9e0d07f8cc3cd1a0e1955a